fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -708,6 +708,25 @@ public void testPutMalformedRDFOnObject() throws IOException {
                 BAD_REQUEST.getStatusCode(), getStatus(put));
     }
 
+    /**
+     * Ensure that graphs cannot be created or replaced for pairtree child resources
+     *
+     * @throws IOException in case of IOException
+     */
+    @Test
+    public void testPutOnPairtree() throws IOException {
+
+        final String objName = getLocation(postObjMethod());
+        final String pairtreeName = objName.substring(serverAddress.length(), objName.lastIndexOf('/'));
+        final String subjectURI = serverAddress + pairtreeName;
+
+        final HttpPut put = putObjMethod(pairtreeName);
```

Comment:
As noted in the JIRA ticket, this test should be something like:
`final HttpPut put = putObjMethod(pairtreeName + "/test");`